web/packages/core/src/current-script.ts

```diff
@@ -0,0 +1,27 @@
+// This must be in global scope because `document.currentScript`
+// works only while the script is initially being processed.
+export let currentScriptURL = "";
+try {
+    if (
+        document.currentScript !== undefined &&
+        document.currentScript !== null &&
+        "src" in document.currentScript &&
+        document.currentScript.src !== ""
+    ) {
+        let src = document.currentScript.src;
+
+        // CDNs allow omitting the filename. If it's omitted, append a slash to
+        // prevent the last component from being dropped.
+        if (!src.endsWith(".js") && !src.endsWith("/")) {
+            src += "/";
+        }
+
+        currentScriptURL = new URL(".", src).href;
+    }
+} catch (e) {
+    console.warn("Unable to get currentScript URL");
+}
+
+export const isExtension = new URL(currentScriptURL).protocol.includes(
```

Comment:
I agree, but since this has been merged already, do you want to submit this change yourself?

Reply:
I'm not sure it's worth opening a PR for a minor thing like this, so it would be nice if relrel could do it as part of his PR. Otherwise, we could do it the next time we edit the web files (well, we have to remember about this 😄).